### ISSUE TITLE
Message: Request textDocument/completion failed with message: Cannot read property 'start' of undefined

### ISSUE THREAD

**Issue by FunctionRun** [joe-re/sql-language-server#12]
VSCode Version: 1.44.2

I open .sql file ，input words ，then OUTPUT throw error

```
[0514/143718.898943:ERROR:node_debugger.cc(50)] Error parsing node options: [DEP0062]: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` and `node --inspect-brk` instead.
start sql-language-server
start sql-language-server
[Error - 2:37:25 PM] Request textDocument/completion failed.
  Message: Request textDocument/completion failed with message: Cannot read property 'start' of undefined
  Code: -32603 
```

**Comment by joe-re** [comment]
@FunctionRun Thank you for reporting! I'll deal with it.

**Comment by joe-re** [comment]
@FunctionRun I've released v0.2.6. Can you update it and try again?
If problem's not solved, can you share me example sql that can repro problem?

**Comment by joe-re** [comment]
I'll mark it complete. If problem is persistence feel free reopen it.
